schemas/com.slack.sauna.commands/send_message/jsonschema/1-0-0

```diff
@@ -0,0 +1,123 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a Slack send message command",
+    "self": {
+        "vendor": "com.slack.sauna.commands",
+        "name": "send_message",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
```

Comment:
Based on https://api.slack.com/docs/messages. There's no documentation that covers all possible webhook fields, so these are just the fields used in various examples.